app/kubemci/cmd/delete.go

```diff
@@ -82,6 +85,7 @@ func addDeleteFlags(cmd *cobra.Command, options *DeleteOptions) error {
 	cmd.Flags().StringSliceVar(&options.KubeContexts, "kubecontexts", options.KubeContexts, "[optional] contexts in the kubeconfig file to delete the ingress from")
 	// TODO(nikhiljindal): Add a short flag "-p" if it seems useful.
 	cmd.Flags().StringVarP(&options.GCPProject, "gcp-project", "", options.GCPProject, "[required] name of the gcp project")
+	cmd.Flags().StringVarP(&options.namespace, "namespace", "n", options.namespace, "[optional] namespace for the ingress")
```

Comment:
same: say it overrides the namespace in ingress?

Reply:
moved to behavior consistent with kubectl